rockets/serverContext.cpp

```diff
@@ -64,8 +64,21 @@ ServerContext::ServerContext(const std::string& uri, const std::string& name,
     if (uvLoop && !uvLoopRunning)
         throw std::runtime_error(
             "provided libuv loop either not valid or not running");
+
+#if LWS_LIBRARY_VERSION_NUMBER >= 3000000
+    // variable has to exists until lws_create_context is done
+    // see minimal-http-server-eventlib-foreign.c
+    void *foreign_loops[1];
     if (uvLoopRunning)
+    {
         info.options |= LWS_SERVER_OPTION_LIBUV;
+        foreign_loops[0] = uvLoop;
+        info.foreign_loops = foreign_loops;
```

Comment:
just wondering if it could be possible to "cheat" with
info.foreign_loops = &uvLoop;

Reply:
possible :)